aries-site/src/pages/components/radiobuttongroup.mdx

```diff
@@ -29,9 +29,11 @@ import {
 
 ### When to use
 
-RadioButtonGroup is used to lay out a discreet list of mutually related options that are easily visible. RadioButtonGroup requires an input choice, so it is important to be clear with what is being asked of the user on the label.
+RadioButtonGroup is used to lay out a clear list of distinct, mutually exclusive options that are easily visible. Since a selection is required, it’s important to be clear with what is being asked of the user on the label.
```

Comment:
```suggestion
RadioButtonGroup is used to lay out a list of distinct, mutually exclusive options that are easily visible. Since a selection is required, it’s important to be clear with what is being asked of the user on the label.
```
Felt like unnecessary word?

Reply:
Also is a selection required?